webapp/components/MapZoneSelector.tsx

```
@@ -0,0 +1,153 @@
+    ].join(" "),
+  );
+
+  return (
+    <Button
+      className={buttonClasses}
+      onClick={props.onClick}
+      value={props.zone}
+    >
+      <div
+        className="w-3 h-3"
+        dangerouslySetInnerHTML={{ __html: svgContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)